code/modules/hydroponics/grown.dm

```diff
@@ -338,7 +355,7 @@
 	reagents.add_reagent("synaptizine", 1 + round(potency / 8, 1))
 	reagents.add_reagent("morphine", 1 + round(potency / 10, 1))
 	reagents.add_reagent("space_drugs", 1 + round(potency / 10, 1))
-	reagents.add_reagent("vitamin", 1 + round((potency / 10), 1))
+	reagents.add_reagent("vitamin", 1 + round((potency / 25), 1))

```

Comment:
Make sure this doesn't cause container overflow. At potency 100, the total reagents should add up to no more than the container limit (50 iirc)